bitnami/spark/templates/statefulset-worker.yaml

```diff
@@ -142,7 +142,7 @@ spec:
           - name: certs
             mountPath: /certs
           - name: shared-certs
-            mountPath: /bitnami/spark/conf/certs
+            mountPath: /opt/bitnami/spark/conf/certs
```

Comment:
Sorry for the inconvenience, but I may have to ask you to also apply my previous suggestion here